lib/parsers/racket-pass-dump-parser.ts

```diff
@@ -0,0 +1,306 @@
+// Copyright (c) 2022, Compiler Explorer Authors
```

Comment:
Can you fix up this to 2023 (not that it's 2023 for much longer)

Reply:
Okay, year updated.